### PR TITLE
[webgpu] Remove tfjs-backend-webgl and tfjs-converter dependency

### PR DESCRIPTION
**Pull request by haoyunfeix**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5428)

### CODE REVIEW

**Comment by haoyunfeix** [comment]
@qjia7 @axinging @xhcao @gyagp PTAL

**Comment by qjia7** [comment]
@haoyunfeix Thanks for the cleaning. Please also change below files: `scripts/package_dependencies.json` remove `"tfjs-converter"`.

File: tfjs-backend-webgpu/package.json [hunk 1]
```diff
@@ -15,13 +15,9 @@
     "bundle": "rollup -c",
     "bundle-ci": "rollup -c --ci",
     "build-link-package": "cd ../link-package && yarn build",
```
Comment (qjia7) [inline tfjs-backend-webgpu/package.json:17]: `yarn build` will also build `yarn build-backend-webgl && yarn build-tflite` in link-package. Maybe we need a new one, which only includes `yarn build-backend-cpu && yarn build-core`.
@lina128 @jinjingforever What's your opinion?

**Review by qjia7 — APPROVED** [review]

**Review by lina128 — APPROVED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5428#-:-MgH15oeA2YqvCR8ldDy:bb6ywmx)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @haoyunfeix, @jinjingforever, and @mattsoulanille)

___
*[tfjs-backend-webgpu/src/backend_webgpu_test.ts, line 250 at r2](https://reviewable.io/reviews/tensorflow/tfjs/5428#-MgH0EiI82tTfpi6V_6p:-MgH0EiJB8vyOakDfgO9:b-hep3se) ([raw file](https://github.com/tensorflow/tfjs/blob/04633433fdb1806fbac58b0db31d0f0313ff3a05/tfjs-backend-webgpu/src/backend_webgpu_test.ts#L250)):*
<details><summary><i alt="summary">Quoted 15 lines of code…</i></summary>

> ```
>   it('should be possible to move data from webgl to webgpu', async () => {
>     tf.setBackend('webgl');
>     const a = tf.randomNormal([1, 65, 65, 256]);
>     const b = tf.randomNormal([1, 65, 65, 256]);
>     const c = tf.add(a, b);
>     await c.data();
>     const f = async () => {
>       tf.setBackend('webgpu');
>       const d = tf.add(a, b);
>       await d.data();
>       expect(() => d.dataSync()).not.toThrow();
>     };
>     await f();
>   });
> });
> ```
</details>

For future reference, we can move this test to e2e, when we add WebGPU dependency there. The goal of e2e is to have comprehensive tests of all the backends. SG removing the test for now.

**Pull request merged**